dev/breeze/src/airflow_ci/sc_get_changed_files.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
```

Comment:
I think we do not need a separate command for "get_changed_files". This is (at least for now) not very useful. But it should be a "utility" function. something that other real "commands" can use.

I think our commands should be centered around the "functionality" we need from selective checks.

Just to explain what I envision:

```
airflow-selective-checks  build-image
```

This should produce one output:

```
::set-output name=image-build::true
```
or
```
::set-output name=image-build::false
```

Depends on the set of changed files.


Similarly we should have other commands:

```
airflow-selective-checks  matrix-strategy
```

This one should produce all the "matrix" components (also as ga-output):
- python, backend, kubernetes etc.

Reply:
All of the above should use "get_changed_files" but as an internal Python method rather than click "command".